 diff --git a/drivers/mmc/core/core.c b/drivers/mmc/core/core.c
-index f1678fd2e..f9c4ccecc 100755
+index 659b5ff09..70ef82cea 100755
 --- a/drivers/mmc/core/core.c
 +++ b/drivers/mmc/core/core.c
-@@ -62,7 +62,7 @@ static const unsigned freqs[] = { 400000, 300000, 200000, 100000 };
+@@ -58,7 +58,7 @@ static const unsigned freqs[] = { 400000, 300000, 200000, 100000 };
   * performance cost, and for other reasons may not always be desired.
   * So we allow it it to be disabled.
   */
@@ -22,15 +43,15 @@ index f1678fd2e..f9c4ccecc 100755
  
  static int mmc_schedule_delayed_work(struct delayed_work *work,
 diff --git a/kernel/module.c b/kernel/module.c
-index 0df5812ae..f815fdc82 100755
+index 83b459f6a..fe36dd65f 100755
 --- a/kernel/module.c
 +++ b/kernel/module.c
-@@ -1345,9 +1345,9 @@ static int check_version(const struct load_info *info,
+@@ -1315,9 +1315,9 @@ static int check_version(const struct load_info *info,
  	return 1;
  
  bad_version:
 -	pr_warn("%s: disagrees about version of symbol %s\n",
-+	pr_warn("%s: disagrees about version of symbol, but ignore...%s\n",
++	pr_warn("%s: disagrees about version of symbol %s\n, but ignore...",
  	       info->name, symname);
 -	return 0;
 +	return 1;